content/en/docs/languages/java/getting-started.md

```diff
@@ -9,23 +9,21 @@ weight: 10
 This page will show you how to get started with OpenTelemetry in Java.
 
 You will learn how you can instrument a simple Java application automatically,
-in such a way that [traces][], [metrics][] and [logs][] are emitted to the
+in such a way that [traces][], [metrics][], and [logs][] are emitted to the
 console.
 
 ## Prerequisites
 
 Ensure that you have the following installed locally:
 
-- Java 17 JDK
+- Java JDK 17+, due to the use of Spring Boot; [Java 8+ otherwise][java-vers]
```

Comment:
(Spring Boot 3 sort of famously made the leap to Java 17)

```suggestion
- Java JDK 17+, due to the use of Spring Boot 3; [Java 8+ otherwise][java-vers]
```

Reply:
Works for me.